lighthouse-core/audits/non-composited-animations.js

```diff
@@ -13,7 +13,7 @@ const UIStrings = {
   title: 'Avoid non-composited animations',
   /** Description of a diagnostic LH audit that shows the user animations that are not composited. */
   description: 'Animations which are not composited can be janky and contribute to CLS. ' +
-    '[Learn more](https://developers.google.com/web/fundamentals/performance/rendering/stick-to-compositor-only-properties-and-manage-layer-count)',
+    '[Learn more](https://web.dev/non-composited-animations)',
```

Comment:
Make sure to run `yarn update:sample-json`

Reply:
Done